Makefile

```diff
@@ -351,6 +372,7 @@ docker-images-only: docker-images-cassandra \
 .PHONY: build-crossdock-binary
 build-crossdock-binary:
 	$(GOBUILD) -o ./crossdock/crossdock-$(GOOS)-$(GOARCH) ./crossdock/main.go
+	sha256sum crossdock/crossdock-$(GOOS)-$(GOARCH) > ./crossdock/crossdock-$(GOOS)-$(GOARCH).sha256sum.txt
```

Comment:
Couldn't we do it in scripts/package-deploy.sh instead of repeating command for every binary?  checksum files are not useful by themselves outside of packaging a release

Reply:
It would require a different repro-check script and would omit anything that isn't currently explicitly packaged for release.

Reply:
I removed all these checksums and changed how this works. I'm actually happier with how this turned out.